src/dbt_client/dbtCoreCommandIntegration.ts

```
@@ -142,7 +142,7 @@ export class DBTCoreCommandProjectIntegration extends DBTCoreProjectIntegration
         (line) =>
           line &&
           line.hasOwnProperty("data") &&
-          line.data.hasOwnProperty("compiled"),
+          line.data?.hasOwnProperty("compiled"),
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)